Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -161,23 +165,52 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   // instantiations
   // (for the moment this assumes the file o2sim_grp.root to be in the current directory)
   const auto grp = readGRP();
-  if (!grp) {
+  if (!grp.get()) {
     return specs;
   }
 
+  // lambda to extract detectors which are enabled in the workflow
+  auto isEnabled = [&configcontext, grp](o2::detectors::DetID id) {
+    auto extractIgnored = [](ConfigContext const& configcontext) {
```

Comment:
Should be fine, but is not it easier to have
```
auto isEnabled = [&configcontext, grp](o2::detectors::DetID id) {
   auto skipString = configcontext.options().get<std::string>("skipDet");
   return (skipString.find(id.getName())==std::string::npos && grp->isDetReadOut(id);
}
```

Reply:
Not sure here.  This would merely look for the presence of a string inside another string? So `--skipDet FOOTPCBAR` would skip the TPC detector ... could work but it does not complain about wrong user input.